CODE_OF_CONDUCT.md

```diff
@@ -1,21 +1,16 @@
-# Code of Conduct
+# The Creator's Code
```

Comment:
Or should that be "The Creator's [c]Ode". (I really don't like "Code of Conduct" as a term - it always makes me think of behaviour management policies at school.)